lib/gui/models/flash-state.js

```diff
@@ -220,3 +220,20 @@ exports.getLastFlashSourceChecksum = () => {
 exports.getLastFlashErrorCode = () => {
   return exports.getFlashResults().errorCode;
 };
+
+/**
+ * @summary Get last flash uuid
+ * @function
+ * @public
+ *
+ * @description
+ * This function returns undefined if there was no last flash.
```

Comment:
Hmmm, looks like `resetState` is _only_ used in `lib/gui/components/flash-error-modal/services/flash-error-modal.js`, which (AFAICT) means the UUID will be cleared if a flash error occurs, but will remain set if the flash completes? Would it be more consistent for a successful flash to delete the UUID too, and then this `@description` could become something like `This function returns undefined if there is no flash operation currently taking place.` which _might_ make more sense?